packages/graphqlgen/src/tests/typescript/basic.test.ts

```diff
@@ -0,0 +1,66 @@
+import { generateCode, parseSchema } from '../../'
+import { join } from 'path'
+import { GraphQLGenDefinition } from 'graphqlgen-json-schema'
+import { buildModelMap } from '../../modelmap'
+
+const relative = (p: string) => join(__dirname, p)
+
+test('basic interface', async () => {
+  const config: GraphQLGenDefinition = {
+    language: 'typescript',
+    schema: relative('../fixtures/interface/schema.graphql'),
+    context: relative('../fixtures/interface/types.ts:Context'),
+    models: {
+      Media: relative('../fixtures/interface/types.ts:Media'),
```

Comment:
It might not be necessary to link the `Media` model, since it is only used for inheritance in typescript.